website_media_size/static/src/js/website_media_size.js

```diff
@@ -1,22 +1,22 @@
 /* Copyright 2018 Onestein
  * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
 
-odoo.define('website_media_size', function(require) {
+odoo.define("website_media_size", function (require) {
     "use strict";
 
-    var web_editor_widget = require('web_editor.widget');
+    var web_editor_widget = require("web_editor.widget");
```

Comment:
Respect quotes choice from original authors, please; at least unless there's anything else to change. The same applies for the rest of the file.

Reply:
a pre eslint change...  will fix it.